docker/production.yml

```diff
@@ -135,14 +135,17 @@ services:
 
   # users service
   users:
+    ports:
+      - ${USERS_PORT}
     restart: unless-stopped
     environment:
       - NODE_ENV=production
-      - USERS_PORT
```

Comment:
I've you're going to use this in line 139, you'll need to include it here.

Reply:
The port is defined in `env.production` and specified when running docker (in `docker-compose.yml`, shouldn't that mean this is line is unnecessary?